backend/src/common/services/gift.ts

```diff
@@ -186,4 +222,26 @@ export default class GiftService {
       teamId
     })
   }
+
+  public initDefaultGifts(teamId: string) {
+    const updateOrInitDefaultGiftsOperations = this.initGifts
+      .map(gift => {
+        return Gift.findOneAndUpdate(
+          {
+            $and:
```

Comment:
do we need &and operator here?
https://docs.mongodb.com/manual/reference/operator/query/and/#examples